iib/web/migrations/versions/eec630370e68_support_deprecation_list_in_add_request_.py

```diff
@@ -0,0 +1,47 @@
+"""Support deprecation_list in Add request type.
+
+Revision ID: eec630370e68
+Revises: 60f89c046096
+Create Date: 2021-01-20 20:36:29.184275
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eec630370e68'
+down_revision = '60f89c046096'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
It would be super if we could add tests for these db migrations in https://github.com/release-engineering/iib/blob/master/tests/test_web/test_migrations.py.